replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/spring/ItemRepository.java

```diff
@@ -0,0 +1,23 @@
+package org.codice.ditto.replication.api.impl.spring;
+
+import java.util.Optional;
+import org.codice.ditto.replication.api.impl.data.ReplicationItemImpl;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+// This will be AUTO IMPLEMENTED by Spring into a Bean called itemRepository
+public interface ItemRepository extends PagingAndSortingRepository<ReplicationItemImpl, String> {
+
+  Optional<ReplicationItemImpl> findByIdAndSourceAndDestination(
```

Comment:
spring lets us create custom query methods intuitively based on the name given. Good examples are in section 2.2.2 here https://docs.spring.io/spring-data/data-solr/docs/1.0.0.RC1/reference/htmlsingle/#solr.query-methods.criterions section 1.3 has a more thorough explanation. Note, the field names here must match the field names of the entity class.